File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        count_map = {}
        for i in range(0, len(nums)):
            if (target - nums[i]) in count_map.keys():
                return [count_map[target - nums[i]], i]
            count_map[nums[i]] = i
            

File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> counts(3, 0);
        for (int i=0; i<nums.size(); i++) {
            counts[nums[i]]++;
        }

        int i = 0;
        for (int n=0; n<counts.size(); n++) {
            for (int j = 0; j<counts[n]; j++) {
                nums[i] = n;
                i++;
            }
        }
    }
};